CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@
 You can do some things to increase the chance that your pull request is accepted the first time:
 
  * Submit one pull request per fix or feature.
- * If your changes are not up to date - rebase your branch on master.
+ * If your changes are not up to date - rebase your branch.
```

Comment:
:+1: done  :)
@sstok I always rebased on some local verion of master, that was fun as well :cold_sweat: :blush: